app/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import Column, String, Boolean, DateTime
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.dialects.postgresql import UUID
+import uuid
+from datetime import datetime
+from app.db.database import Base
+from pydantic import BaseModel, EmailStr, Field
+from datetime import datetime
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id: Mapped[uuid.UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid.uuid4
+    )
+    username: Mapped[str] = mapped_column(String, nullable=False)
+    email: Mapped[str] = mapped_column(String, unique=True, nullable=False)
+    hashed_password: Mapped[str] = mapped_column(String, nullable=False)
+    is_admin: Mapped[bool] = mapped_column(Boolean, default=False)
+    is_active: Mapped[bool] = mapped_column(Boolean, default=True)
+    created_at: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow)
+    updated_at: Mapped[datetime] = mapped_column(
```

Comment:
The default value for the update date should be null.

Reply:
I did all of the things , but not this  , can i get a hint for it because make the value null will lead a lot of bugs .

Reply:
What are the bugs?

Reply:
if there is any bug regarding having the update date as null, you should fix it

Reply:
Done.
the problem is the database can't accept a null value for update, I tried everything, and the solution was to alter the table manually, I did from Dbeaver SQL commands.

Reply:
This is most likely because you're using a Docker volume. Whenever you start the service, the table will already be present, and the CREATE TABLE IF NOT EXISTS will only create the table if it doesn't exist. So, next time, you can delete the volume folder or not use volume for now.